recipes/gempy_engine/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "gempy_engine" %}
+{% set version = "2024.2.0" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 35bd3e764305fd49955713703913e4619e12a7780439f342c0616c2c361e335a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - numpy >=1.23
```

Comment:
Upstream only says `numpy`. Is there a reason to constrain the version?

Reply:
Not really, i corrected it